pkg/kubelet/container/runtime.go

```diff
@@ -176,6 +176,8 @@ type RunContainerOptions struct {
 	// into docker's container runtime.
 	NetMode string
 	IpcMode string
+	// The parent cgroup to pass to Docker
+	CgroupParent string
```

Comment:
I am OK with this. But seems we need to change the comment to remove 'Docker'. My understanding is you don't want other container runtime to escape the cgroup as well, right? @guenter